### PR TITLE
Pass cert and password parameters to hammer-brave.bat

### PR DESCRIPTION
**Pull request by mbacchi**
Fixes https://github.com/brave/brave-browser/issues/1990

### CODE REVIEW

**Review by w0ts0n — DISMISSED** [review]
looks good

File: build_omaha.py [hunk 1]
```diff
@@ -61,7 +72,9 @@ def Tagging(args, omaha_dir, debug):
   omaha_out_dir = os.path.join(omaha_dir, 'omaha', 'scons-out', last_win_dir)
   apply_tag_exe = os.path.join(omaha_out_dir, 'obj', 'tools', 'ApplyTag', 'ApplyTag.exe')
 
-  tag = 'appguid=APP_GUID&appname=TAG_APP_NAME&needsadmin=prefers&lang=en&ap=TAG_AP'
+  # right now the needsadmin flag is hardcoded, but should be configurable with an env var or command line flag
+  tag = 'appguid=APP_GUID&appname=TAG_APP_NAME&needsadmin=False&lang=en&ap=TAG_AP'
```
Comment (simonhong) [inline build_omaha.py]: Please check that `False` will be only used for migration from muon to brave-core.
Normally, it should be `prefers`. @bsclifton

File: build_omaha.py [hunk 2]
```diff
@@ -20,6 +20,17 @@ def Build(args, omaha_dir):
   # move to omaha/omaha and start build.
   os.chdir(os.path.join(omaha_dir, 'omaha'))
   command = ['hammer-brave.bat', 'MODE=all', '--all']
```
Comment (simonhong) [inline build_omaha.py]: nit: This line can be deleted.

File: build_omaha.py (hunk 1 above)
Comment (mbacchi) [reply build_omaha.py]: I updated that section to use 'prefers' as the default. We will set the env var TAG_ADMIN in Jenkins for the release build for the migration from browser-laptop only. @bsclifton FYI

File: build_omaha.py (hunk 2 above)
Comment (mbacchi) [reply build_omaha.py]: Removed. An oversight on my part.

File: build_omaha.py [hunk 3]
```diff
@@ -99,6 +113,8 @@ def ParseArgs():
                       nargs=1)
   parser.add_argument('--tag_ap',
                       nargs=1)
+  # parser.add_argument('--tag_admin',
```
Comment (simonhong) [inline build_omaha.py]: nit: This would not be used because you introduced `TAG_ADMIN` env.

**Review by simonhong — DISMISSED** [review]
LGTM with nits.

**Review by simonhong — APPROVED** [review]
LGTM

File: build_omaha.py (hunk 3 above)
Comment (mbacchi) [reply build_omaha.py]: Removed, I'll merge as soon as you approve again.

**Pull request merged**